PathOfTerraria.csproj

```diff
@@ -17,6 +17,9 @@
     <BuildCommand Condition=" Exists('/etc/NIXOS') ">steam-run dotnet</BuildCommand>
     <BuildCommand Condition=" '$(BuildCommand)' == '' ">dotnet</BuildCommand>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
Ur fav

Reply:
Yet every time I'm like, a nice, simple PR. Should be straight to merge